src/store/types.ts

```diff
@@ -1,14 +1,15 @@
 import { CommitOptions, DispatchOptions, Store as VuexStore } from 'vuex';
 import { AxiosError, AxiosResponse } from 'axios';
 import { CooperationStateInterface } from '@/store/cooperation';
+import { UserStateInterface } from '@/store/user';
 import { Actions, Getters, Mutations } from '@/store/cooperation/types';
 
 export interface RootStateInterface {
   hello: string;
   world: string;
 }
 
-export type Store = CooperationStore<CooperationStateInterface>;
+export type Store = CooperationStore<CooperationStateInterface> & UserStore<UserStateInterface>;
```

Comment:
maybe will be better
export interface StoreInterface {
  cooperationStore: CooperationStateInterface,
  userStore: UserStateInterface
}